### DISCUSSION TITLE
Filtering with a column in another table

### DISCUSSION THREAD

**Discussion by zikook** [biolab/orange3#6388] [Q&A] [answered]
Hi,
I am trying to return rows in a table that intersect on column 'mirfam' with another table (example files below).

table 1:
**mirfam**
mirfam1
mirfam2

table 2:
**cell    mirfam**
cell1    mirfam1
cell1    mirfam2
cell2    mirfam2
cell3    mirfam4

The output data table should be:
**plant    mirfam**
cell1    mirfam1
cell1    mirfam2
cell2    mirfam2

I have been unsuccessful with both the Venn diagram and Merge Data widgets. 
How can I accomplish this?
Thanks in advance,
-Aaron

**Comment by janezd** [comment, marked as answer]
Can you add another, dummy column to table 1? For instance, add `x` whose values is always 1. Or whatever.

Then you can use Merge Data; pass table 2 as Data and table 1 as Extra Data. Column `x` will contain 1 for those rows in table 2 that are matched by table 1, and missing values (?) for those that are not. Then add Select Columns with condition "x is defined" :)

There may be a better way, but that's the first that came to my mind.

**Reply by zikook** [reply]
Brilliant! Thanks!
